### DISCUSSION TITLE
Kocham ten program

### DISCUSSION THREAD

**Discussion by PyczekFromPoland** [TowarzyszFatCat/doccli#13] [General]
Zawsze zastanawiałem się jak mogłbym szybko oglądać anime po polsku bez wchodzenia na jakieś stronki, i nagle znajduje to cudo! Nie dość że na linux'a to jeszcze przypomina program w którym oglądałem anime do tego czasu (ani-cli), naprawde szkoda patrzeć że ten projekt ma tak mało gwiazdek. 

TL;DR dzieki za ten zajebisty program :D

- Pyczek

**Comment by TowarzyszFatCat** [comment]
Dzięki wielkie za miłe słowa. Miałem taki sam problem jak Ty, dlatego napisałem doccli :)

**Comment by mareczek82** [comment]
Dołączam się do podziękowań, korzystanie z doccli jest czystą przyjemnością i bardzo przystępne nawet dla mało zaawansowanego użytkownika jak ja. 

@TowarzyszFatCat Towarszyszu, czy planujesz dodanie opcji pobierania? Mieszkam w miejscu ze słabym łączem wiec taka opcja pobierania na później byłaby rewelacyjna.

**Reply by TowarzyszFatCat** [reply]
**Dzięki @mareczek82!**

> @TowarzyszFatCat Towarszyszu, czy planujesz dodanie opcji pobierania? Mieszkam w miejscu ze słabym łączem wiec taka opcja pobierania na później byłaby rewelacyjna.

Co do opcji pobierania, planuje ją dodać. Coś w stylu "biblioteki offline", gdzie będzie można pobierać całe serie lub poszczególne odcinki. Tyle że wymaga to ode mnie lekkiego przerobienia programu. Być może niedługo będę przepisywał doccli od nowa, więc dodanie takiej opcji nie będzie problemem.

**Reply by mareczek82** [reply]
Fantastyczna wiadomość, trzymam kciuki :)

**Comment by Tropical2497** [comment]
polecam sprawdzić projekt:https://github.com/Benexl/FastAnime?tab=readme-ov-file
może się nim "zainspirujecie"

**Reply by TowarzyszFatCat** [reply]
Dokładnie z niego jest inspiracja na nowe menu i integracje z anilistą. Możesz sobie zerknąć do tego brancha:  https://github.com/TowarzyszFatCat/doccli/tree/v3-rewrite . Ale jak na razie największym problemem jest brak czasu na rozwój projektu :/

**Reply by mareczek82** [reply]
@TowarzyszFatCat pamiętaj o opcji pobierania ;)

trzymam kciuki za twój projekt, jeszcze raz dzięki
